tokio-postgres/src/connection.rs

```diff
@@ -343,6 +336,26 @@ where
             },
         }
     }
+
+    /// Polls for asynchronous messages from the server.
+    ///
+    /// The server can send notices as well as notifications asynchronously to the client. Applications that wish to
+    /// examine those messages should use this method to drive the connection rather than its `Future` implementation.
+    ///
+    /// Return values of `None` or `Some(Err(_))` are "terminal"; callers should not invoke this method again after
+    /// receiving one of those values.
+    pub fn poll_message(
```

Comment:
Why this change was needed? It doesn't look like it's used in the new code

Reply:
It is used. It replaces the original "poll_message", which was renamed just above